src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// Write code here
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+import './Person.scss';
+
+export function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  const partner = (sex === 'm')
+    ? (`My wife's name is ${partnerName}`)
+    : (`My husband's name is ${partnerName}`);
```

Comment:
you can shorten it, just by doing ternary inside a string, for the word `wife/husband`